doc/api/assert.md

```diff
@@ -26,12 +29,33 @@ When using the `strict mode`, any `assert` function will use the equality used i
 the strict function mode. So [`assert.deepEqual()`][] will, for example, work the
 same as [`assert.deepStrictEqual()`][].
 
+Besides that error messages that involve objects produce a error diff instead of
```

Comment:
Nit: `a errror diff` -> `an error diff`

The sentence seems to be missing some punctuation for clarify. Probably a comma after the second word?